xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -29,18 +29,32 @@
 ## Title and parent
 ##
 <div id="parentinput" class="form-group">
-  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
+  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') 
+    <a id='hideEditParentTrigger' 
+     href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
```

Comment:
That's not he way you should format HTML. Why not simply:

```
<label ...>
  <a ...>
    ...
  </a>
</label>
```

But this formatting is out of the scope of this PR. It's probably better to keep only the changes that are relevant for the issue you're trying to fix.

Reply:
But it is over 120 chars, should I format it or keep it as is?

Reply:
Looks good, but I would prefer to keep in this PR only the changes that are relevant for the current issue. You can make later another PR to fix the formatting.

Reply:
Ok, let me do this quickly.